src/templates/cart/voucher.template.html

```diff
@@ -213,7 +213,6 @@ <h4>#[@item_count@] &nbsp; [@vprogram_title@] -
 <!--##[%END DATA%]##-->
 </div>
 <br clear="all">
-</div>
```

Comment:
@dalerodgie that closing div is actually needed, but the DOM is incorrect, it should be after the closing form tag. It is for: https://github.com/NetoECommerce/Skeletal/blob/master/src/templates/cart/voucher.template.html#L1

Can you please update this on your PR?